src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -899,14 +899,14 @@ export class DefaultStyleController implements IStyleController {
 
 		if (styles.listFocusAndSelectionBackground) {
 			content.push(`
-				.monaco-drag-image,
+				.monaco-drag-image${suffix},
```

Comment:
Does `.monaco-drag-image` with a `suffix` make any difference? Whats the intent?

Reply:
Each list has its own styles. explorer might have `listFocusAndSelectionBackground` set to red but outline could have `listFocusAndSelectionBackground` set to red. Suffix is needed to make sure the style applies only to that specific tree/list and not all (this was a bug before)

Reply:
But does this work? I see it makes sense for `monaco-list`, but the drag image is root leveled and not within the suffix?

Reply:
Yes, it works, I tested it. `applyDragImage` supports adding a extraClass to the dragImage. The listWidget uses this and adds its suffix so the styles will be applied correctly